src/bigartm/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ include_directories(${CMAKE_CURRENT_LIST_DIR}/../)
 
 set(SRC_LIST
```

Comment:
Sorry. I don't know where I should add my header in order to use class from it in srcmain. I thought it's common.h until I looked into it.